middlewares/verify-signup.js

```diff
@@ -0,0 +1,16 @@
+const { User } = require("../models/user-model")
+
+checkDuplicateUsername = async (req, res, next) => {
+    try{ 
```

Comment:
formatting of code can be consistent with spacing

Reply:
will do, thankyou!